scripts/travis-ci/run-test.sh

```diff
@@ -12,7 +12,10 @@ case "$1" in
         ;;
     *)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
+        composer install
         cd $DRUPAL_DIR
+        php modules/composer_manager/scripts/init.php
```

Comment:
I need it to update drupal autoload so when I test the integration I won't failed.